test/compilable/test19891.d

```diff
@@ -0,0 +1,7 @@
+void fun(R)(auto ref int a, auto ref R r = 1) {}
+
+void main()
```

Comment:
Don't use main() in a compile-only test

Reply:
Is there any particular reason for that? it's already used in quite many `compilable/` tests.